src/components/Request/Pay/Pay.consts.ts

```diff
@@ -9,7 +9,13 @@ export interface IPayScreenState {
     idx: number
 }
 
-export type IRequestLinkState = 'LOADING' | 'READY_TO_PAY' | 'ALREADY_PAID' | 'ERROR' | 'CANCELED'
+export enum IRequestLinkState {
+    LOADING = 'LOADING',
+    READY_TO_PAY = 'READY_TO_PAY',
+    ALREADY_PAID = 'ALREADY_PAID',
+    ERROR = 'ERROR',
+    CANCELED = 'CANCELED'
+}
```

Comment:
This now conflicts with https://github.com/peanutprotocol/peanut-ui/pull/443

@jjramirezn can you review this?

Reply:
For things like state I like enum vs a string type. I don't think there is any conflict #443 is merged to develop already (and the changes I made were over a particular view)